dlp/patchTrigger.js

```diff
@@ -0,0 +1,74 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// sample-metadata:
+//  title: Update job trigger with suitable configuration and mask.
+//  description: Uses the Data Loss Prevention API to update job trigger with suitable configuration and mask.
+//  usage: node patchTrigger.js projectId, jobTriggerName
+function main(projectId, jobTriggerName) {
+  // [START dlp_update_trigger]
+  // Imports the Google Cloud Data Loss Prevention library
+  const DLP = require('@google-cloud/dlp');
+
+  // Instantiates a client
+  const dlpClient = new DLP.DlpServiceClient();
+
+  // The project ID to run the API call under
+  // const projectId = 'my-project';
+
+  // The job trigger ID to run the API call under
+  // const jobTriggerName = 'your-job-trigger-name';
+
+  async function patchTrigger() {
```

Comment:
Can we call this update to stay consistent with the rest of the example?
Same question regarding filename